test/e2e/serial/tests/version.go

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package tests
+
+import (
+	"context"
+	"encoding/json"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	nropv1 "github.com/openshift-kni/numaresources-operator/api/v1"
+	"github.com/openshift-kni/numaresources-operator/pkg/version"
+
+	"github.com/openshift-kni/numaresources-operator/internal/api/buildinfo"
+	"github.com/openshift-kni/numaresources-operator/internal/remoteexec"
+
+	e2eclient "github.com/openshift-kni/numaresources-operator/test/internal/clients"
+	"github.com/openshift-kni/numaresources-operator/test/internal/deploy"
+	"github.com/openshift-kni/numaresources-operator/test/internal/objects"
+)
+
+var _ = Describe("[serial] numaresources version", Serial, Label("feature:config"), func() {
+	When("starting the configuration suite", func() {
```

Comment:
`s/configuration/serial/`?

Reply:
uhm, probably a complete rephrase is better